Src/Middlewares/ValidatorMiddleware.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response, NextFunction } from "express";
+import { InvalidBodyParameterException } from "../Exceptions/InvalidParamsException";
+
+export const validateRequiredParams = (requiredParams: string[]) => {
+	return (req: Request, res: Response, next: NextFunction) => {
+		try {
```

Comment:
You don't need to wrap these in a try catch block since you have this line `app.use(errorHandler);` in the `src/index`. Just throw the exception and let the global handler handle it.